### DISCUSSION TITLE
Handle additions/deletions within bill text.

### DISCUSSION THREAD

**Discussion by jeremybmerrill** [openstates/enhancement-proposals#4] [Ideas]
e.g. https://legis.delaware.gov/json/BillDetail/GenerateHtmlDocument?legislationId=26010&legislationTypeId=1&docTypeId=2&legislationName=HB248ocd-bill/5793a6bb-6dcd-496d-b970-419568789932 has strikethrough'ed text (by CSS class, frustratingly) and underlined text to indicate deletions and additions, respectively. What do we want to extract? I'm thinking we would exclude the deletions from our conception of "bill text", but include the existing law (for context, so the result is semi-readable) and the additions. What do you all think?

**Comment by jeremybmerrill** [comment]
Florida too: http://flsenate.gov/Session/Bill/2017A/6A/BillText/er/HTML

**Comment by jeremybmerrill** [comment]
It would increase complexity, but another options is we could output one text file that includes the deletions in the bill text and another that excludes them.

I can see us wanting to include the deletion text if this is, for instance, feeding ElasticSearch. A bill that deletes a word from statute should still come up in a search, for instance. But for machine-learning purposes, it might not...

**Comment by jeremybmerrill** [comment]
In some cases the formatting may be amended bill versions.

In other cases it may be a "diff" of a statute. 

We may want to treat these differently.

**Comment by jamesturk** [comment]
This is still unresolved, since in most cases it will result in a good amount of added complexity.  I'm all for tackling this but don't have the bandwidth/need for now.  If someone comes along & does, I'd be more than glad to discuss options!
